en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md

```diff
@@ -0,0 +1,157 @@
+    
+    1. Navigate to [Microsoft Dynamics 365 account](https://portal.office.com/) and select **Create a Dynamics 365  Account**.
+    
+    2. Login to created **Dynamics 365 account**. Click **Admin** icon.
+    
+       <img src="/assets/img/connectors/MSdynamics365-admin-icon.png" title="Dynamics365 admin center" width="800" alt="Dynamics365 admin center"/>
+    
+    3. Click **Show all** from the dropdown in the left corner scroll bar.
+    
+       <img src="/assets/img/connectors/MSdynamics365-show-all.png" title="Dynamics365 show all" width="800" alt="Dynamics365 show all"/>
+       
+    4. Click **All admin centers** and click **Dynamics 365** icon. It will navigate to the **Power Platform admin center**
+
+       <img src="/assets/img/connectors/MSdynamics365-all-admin-center.png" title="Dynamics365 all admin center" width="800" alt="Dynamics365 all admin center"/>
+
+    5. Select created environment and click **Open environment**.
+    
+       <img src="/assets/img/connectors/MSdynamics365-select-environment.png" title="Dynamics365 select environment" width="800" alt="Dynamics365 select environment"/>
+       
+    6. Navigate to Settings -> Security -> Users.
+    
+       <img src="/assets/img/connectors/MSdynamics365-settings.png" title="Dynamics365 settings" width="800" alt="Dynamics365 settings"/>
+        
+       <img src="/assets/img/connectors/MSdynamics365-Users.png" title="Dynamics365 users" width="800" alt="Dynamics365 users"/>
+    
+    7. Choose **Application Users** in the view filter.
+    
+       <img src="/assets/img/connectors/MSdynamics365-filter-application-user.png" title="Filter application users" width="800" alt="Filter application users"/>
+    
+    8. Select -> **New**.
+       
+    9. In the Application User form, enter the required information.
+    
+       <img src="/assets/img/connectors/MSdynamics-new-application-user.png" title="Dynamics365 new application user" width="800" alt="Dynamics365 new application user"/>
+
+       The user name information must not match a user that exists in the Azure Active Directory.
+       In the **Application ID** field, enter the application ID of the app you registered earlier in the Azure AD.
+       
+    10. If all goes well, after selecting SAVE, the Application ID URI and Azure AD Object Id fields will auto-populate with correct values. 
+    
+       <img src="/assets/img/connectors/MSdynamics365-created-application-user.png" title="Dynamics365 created application user" width="800" alt="Dynamics365 created application user"/> 
+    
+    11. Before exiting the user form, choose MANAGE ROLES and assign a security role to this application user so that the application user can access the desired organization data.
+   
+## Generate Access Token 
+
+After setup Azure and Microsoft Dynamics 365 CRM configurations you can get access token by invoking the following HTTP request.
+
+POST URL:
+
+https://login.microsoftonline.com/<tenant id> /oauth2/token
```

Comment:
What is `tenant id`? 
When you have `<` and `>` how does it appear in the docs?

Reply:
In the above steps already describe how to get the Tenant id.

Reply:
Using "<>" in the docs need to verify.